run_model-dep-full_limits.sh

```diff
@@ -117,39 +109,37 @@ if [[ $MODE == "initial" ]]; then
     if [[ $ANALYSISTYPE == "classic" ]]; then
         morph_parallel.py --output ${defaultdir}/datacards \
             --analysis ${analysis} \
+            --sub-analysis ${sub_analysis} \
+            --categorization ${categorization} \
+            --sm-like-hists ${sm_like_hists} \
+            --sm-gg-fractions ${CMSSW_BASE}/src/CombineHarvester/MSSMvsSMRun2Legacy/data/higgs_pt_reweighting_fullRun2_v2.root \
             --eras 2016,2017,2018 \
-            --category_list ${CMSSW_BASE}/src/CombineHarvester/MSSMvsSMRun2Legacy/input/mssm_classic_categories.txt\
+            --category-list ${CMSSW_BASE}/src/CombineHarvester/MSSMvsSMRun2Legacy/input/mssm_classic_categories.txt \
             --variable mt_tot_puppi \
             --parallel 10 2>&1 | tee -a ${defaultdir}/logs/morph_mssm_log.txt
-
-        morph_parallel.py --output ${defaultdir}/datacards \
```

Comment:
To be allowed to remove this also the file `${CMSSW_BASE}/src/CombineHarvester/MSSMvsSMRun2Legacy/input/mssm_classic_categories.txt` has to be adapted.

Reply:
this is already so in the file:
https://github.com/ArturAkh/MSSMvsSMRun2Legacy/blob/master/input/mssm_classic_categories.txt#L11

Reply:
Then I might have missed it when it has been updated. Sorry for that.